pandas/tests/frame/test_set_index_preserve_dtype.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import pandas as pd
+
+
+@pytest.mark.parametrize('test_dtype', [object, 'int64'])
+def test_dtypes(test_dtype):
```

Comment:
don’t make a new file we have loads of tests in alter_axes u can just add it therr

Reply:
Ok will do